testsuite/python/engine_lb.py

```diff
@@ -84,13 +84,6 @@ def tearDown(self):
         self.system.actors.clear()
         self.system.thermostat.turn_off()
 
-    def test_conflicting_parameters(self):
```

Comment:
why is this test gone?

Reply:
It has been improved and moved there:
https://github.com/espressomd/espresso/blob/606247de636ea8e9921fab6b3cd883727f8be709/testsuite/python/particle.py#L371-L375